code/API_definitions/geofencing.yaml

```diff
@@ -342,6 +349,33 @@ components:
           longitude: 7.10066
         radius: 50000
 
+    Polygon:
```

Comment:
For coherence, we should use same schema that we already have defined for location-retrieval: https://github.com/camaraproject/DeviceLocation/blob/main/code/API_definitions/location-retrieval.yaml#L289, which is very similar, but uses `boundary` and `maxItems` is 15

Reply:
Thanks. The change is done.

Reply:
Thanks, we have to conclude the discussion in the issue #194  before deciding on the PR